tests/test_uploader.py

```diff
@@ -366,6 +366,11 @@ def test_sorted_requirement(req, expected):
     assert uploader._sorted_requirements(req) == expected
 
 
+def test_normalize_requirement():
+    assert ["tf-yarn", "typing-extension", "to-to"] == \
+        uploader._normalize_pex_requirements(["tf_yarn", "typing_extension", "to-to"])

```

Comment:
to-to should be to_to shouldn't it ?

Reply:
No it is part of the test that all "_" and "-" should become "-". I already test that "_" become "-" via "tf_yarn", "typing_extension"

Reply:
yup, I read the test backwards ...